src/twisted/__init__.py

```diff
@@ -7,16 +7,7 @@
 Twisted: The Framework Of Your Internet.
 """
 
-def _checkRequirements():
-    # Don't allow the user to run a version of Python we don't support.
-    import sys
-
-    version = getattr(sys, "version_info", (0,))
-    if version < (2, 7):
-        raise ImportError("Twisted requires Python 2.7 or later.")
-    elif version >= (3, 0) and version < (3, 3):
-        raise ImportError("Twisted on Python 3 requires Python 3.3 or later.")
-
+from twisted._checkrequirements import _checkRequirements
```

Comment:
can we move this to _setup.py and not call each time we import twisted? ... and then we don't need the extra exec code in setup.py.

Reply:
OK, I moved the check into _setup.py